Gems/Atom/RHI/Code/Source/RHI/Scope.cpp

```diff
@@ -30,6 +31,28 @@ namespace AZ
             AZ_Assert(IsInitialized() == false, "Scope was previously initialized.");
             SetName(scopeId);
             m_id = scopeId;
+
+            if (Validation::IsEnabled())
+            {
+                // Cache the string views for better gpu marker verbosity
+                size_t firstDotPos = m_id.GetStringView().find_first_of(".");
+                if (firstDotPos == AZStd::string::npos)
+                {
+                    m_markerWithPipelineName = m_id.GetStringView();
+                    m_marker = m_id.GetStringView();
+                }
+                else
+                {
+                    // Strip the first token before the first "." to extract the marker label with pipeline name
+                    m_markerWithPipelineName = m_id.GetStringView().substr(firstDotPos + 1, m_id.GetStringView().length() - firstDotPos + 1);
+
+                    // Strip the second token before "." to extract the marker label without pipeline name
+                    firstDotPos = m_markerWithPipelineName.find_first_of(".");
+                    m_marker = firstDotPos != AZStd::string::npos ? m_markerWithPipelineName.substr(firstDotPos + 1, m_markerWithPipelineName.length() - firstDotPos + 1)
+                        : m_markerWithPipelineName;
+                }
+            }
+
```

Comment:
I'm not a fan of having to know that the pipeline name is before the first dot, and the rest of the string is the marker name. Can we change it to not put the pipeline name in the scope name if there is only one render pipeline rather than have everyone parse it back out?